### DISCUSSION TITLE
MOP subscription requests denied

### DISCUSSION THREAD

**Discussion by ssrkr** [streamnative/mop#1112] [Q&A]
Hi,
We upgraded MoP and after that we are consistently facing "All subscription requests were denied" on subscription request. What configuration is missing? We are not using authentication or even TLS. Our MOP configuration is the minimal set:
 messagingProtocols=mqtt
 protocolHandlerDirectory=./protocols
 mqttListeners=mqtt://127.0.0.1:1883
 advertisedAddress=127.0.0.1

Can anyone help in fixing the issue? Here is a sample run of mosquitto_sub:

mosquitto_sub -h 127.0.0.1 -d -t "local/mq"
Client (null) sending CONNECT
Client (null) received CONNACK (0)
Client (null) sending SUBSCRIBE (Mid: 1, Topic: local/mq, QoS: 0, Options: 0x00)
Client (null) received SUBACK
Subscribed (mid: 1): 128
Client (null) sending DISCONNECT
All subscription requests were denied.

>> **Subscribed (mid: 1): 128** suggests access is denied but we are not using any access control for MQTT.

**Comment by mattisonchao** [comment]
Hi, which pulsar version are you using?

**Comment by ssrkr** [comment]
This was a little old request, you may close it.
________________________________
